app/forms/state_file/name_dob_form.rb

```diff
@@ -26,6 +26,8 @@ class NameDobForm < QuestionsForm
     validates_presence_of :hoh_qualifying_person_name, if: -> {  @intake.class == StateFileAzIntake && @intake.requires_hoh_qualifying_person_name? }
     validate :primary_birth_date_is_valid_date, if: -> { @intake.ask_primary_dob? }
     validate :spouse_birth_date_is_valid_date, if: -> { @intake.ask_spouse_dob? }
+    validates :primary_first_name, format: { with: /\A[a-zA-Z]{1}([A-Za-z-\s']{0,15})\z/.freeze }
```

Comment:
Do we not permit names that are longer than 16 chars?

Reply:
The pattern is actually from the IRS - and there are a lot of things it does not support (Spanish characters for example - "José" will have to be written as "Jose".)

This is actually in the XSD - so even if we relaxed this constraint, it would just fail when the IRS validate:
<img width="780" alt="image" src="https://github.com/codeforamerica/vita-min/assets/17094895/177806b3-9d3f-40b7-9828-2909962c8394">